app/jobs/reports/irs_verification_demographics_report.rb

```diff
@@ -76,8 +76,8 @@ def issuers
     end
 
     def emails
-      internal_emails = [*IdentityConfig.store.irs_verification_report_config]
-      irs_emails = [] # Need to add IRS email config
+      internal_emails = [*IdentityConfig.store.team_daily_reports_emails]
+      irs_emails = [*IdentityConfig.store.irs_verification_report_config]
```

Comment:
unless there's something else in `**report_config` replace the parameter with `irs_verification_report_emails`